sdks/python/apache_beam/examples/wordcount_it_test.py

```diff
@@ -43,9 +43,11 @@ def test_wordcount_it(self):
     output = '/'.join([test_pipeline.get_option('output'),
                        test_pipeline.get_option('job_name'),
                        'results'])
+    sleep_secs = int(test_pipeline.get_option('sleep_secs'))
```

Comment:
`sleep_secs` should be optional for `wordcount_it_test`. You are right, this will break test if no `sleep_secs` specified. Will fix